src/coprocessor/Client.cc

```diff
@@ -382,11 +395,13 @@ std::vector<BatchCopTask> buildBatchCopTasks(
                 msg += " store " + task.store_addr + ": " + std::to_string(task.region_infos.size()) + " regions,";
             return msg;
         };
-        if (log->getLevel() >= Poco::Message::PRIO_DEBUG)
-            log->debug(tasks_to_str("Before region balance:", batch_cop_tasks));
+        if (log->getLevel() >= Poco::Message::PRIO_INFORMATION)
```

Comment:
Is this necessary?
Seems this judgement exist in `log->information`

Reply:
Guess because to avoid run `tasks_to_str` when log level is warn.